package.json

```diff
@@ -19,23 +19,18 @@
   "license": "MIT",
   "files": ["dist", "src"],
   "source": "src/index.ts",
-  "module": "./dist/index.module.js",
   "main": "./dist/index.cjs",
   "types": "./dist/index.d.ts",
   "exports": {
     ".": {
-      "import": {
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.module.js"
-      },
-      "require": {
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.cjs"
-      }
-    }
```

Comment:
Unless you're shipping separate types for CJS and ESM (which Microbundle won't generate, though I don't think your interface needs it anyhow), there's no reason to nest like this.